sormas-ui/src/main/java/de/symeda/sormas/ui/samples/SampleGridComponent.java

```diff
@@ -106,12 +106,6 @@ public HorizontalLayout createFilterBar() {
 			if(!samplesView.navigateTo(criteria, false)){
 				filterForm.updateResetButtonState();
 				grid.reload();
-
```

Comment:
Please also removed the no longer needed string from the properties file and the getFirstItem() method from the FilteredGrid class if it's no longer used elsewhere

Reply:
ok, done